renderdoc/driver/gl/wrappers/gl_draw_funcs.cpp

```diff
@@ -709,10 +709,47 @@ bool WrappedOpenGL::Serialise_glDrawArrays(GLenum mode, GLint first, GLsizei cou
   return true;
 }
 
+void WrappedOpenGL::CopyClientMemoryArrays(GLint first, GLsizei count)
+{
```

Comment:
No it's fine to leave it as it is. I just had a brain fart and completely missed that bindng the fake VBO meant we'd have to copy just to keep things valid during idle.

I think the early-out for when we have a VAO bound or if we're reading would still be nice, mostly for the former case so we don't have the extra loop running per-drawcall on the majority of cases where we don't need this path.